templates/strings/flat-swift2.stencil

```diff
@@ -4,8 +4,15 @@
 import Foundation
 
 // swiftlint:disable file_length
-{% macro parametersBlock types %}{% for type in types %}p{{forloop.counter}}: {{type}}{% if not forloop.last %}, {% endif %}{% endfor %}{% endmacro %}
-{% macro argumentsBlock types %}{% for type in types %}p{{forloop.counter}}{% if not forloop.last %}, {% endif %}{% endfor %}{% endmacro %}
+{% macro parametersBlock types %}{% filter removeNewlines:"false" %}
+{% for type in types %}
```

Comment:
Side note: I really wish Stencil had whitespace control like ejs & Liquid & other template engines, so that we could write `  {%- for type in types -%}` and let the template engine remove leading whitespaces when we use the dash on the opening brackets of tags. I think Kyle still intends to add it for Stencil 1.0 but it's still not in yet?

If Stencil plans to add it soon we can wait, if Kyle thinks he'll need more time for that we could imagine using a simplified version of it (replacing `^\s*{%-` with `{%` in the template text before passing it to Stencil), a bit hacky but would work in the meantime. Only worth it if whitespace control isn't planned to be added too Stencil proper anytime soon though, imho :wink: